multi_block_core.dic

```diff
@@ -147,6 +147,24 @@ save_cell.diffrn_id
 
 save_
 
+save_cell.structure_id
+
+    _definition.id                '_cell.structure_id'
+    _definition.update            2023-10-16
+    _description.text
+;
+    A code identifying the structure to which this atom site belongs.
+;
```

Comment:
Not 'atom site' but 'cell'. "A code identifying the crystallographic structure associated with this cell description."